apptools/apptools-android-tests/apptools/Check_App_Icon.html

```diff
@@ -0,0 +1,75 @@
+  may be used to endorse or promote products derived from this work without
+  specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY INTEL CORPORATION "AS IS"
+AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ARE DISCLAIMED. IN NO EVENT SHALL INTEL CORPORATION BE LIABLE FOR ANY DIRECT,
+INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
+NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE,
+EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+Authors:
+        Yun, Liu<yunx.liu@intel.com>
+
+-->
+
+<meta charset="utf-8">
+<title>Check_App_Icon</title>
+<link rel="author" title="Intel" href="http://www.intel.com">
+<body>
+  <p>
+    <strong>Test steps:</strong>
+  </p>
+  <ol>
+    <li>If org.xwalk.test exists in apptools-android-tests/tools/ directory, remove it <br/>
+        Then run below commands under apptools-android-tests/tools/ directory: <br/>
+        On Android: <br/>
+        &nbsp; &nbsp; Shared Mode: <br/>
+        &nbsp; &nbsp; &nbsp; &nbsp; $ ./crosswalk-app-tools/src/crosswalk-app create org.xwalk.test --android-shared <br/>
+        &nbsp; &nbsp; Embedded Mode: <br/>
+        &nbsp; &nbsp; &nbsp; &nbsp; $ ./crosswalk-app-tools/src/crosswalk-app create org.xwalk.test <br/>
+        &nbsp; &nbsp; $ cd org.xwalk.test <br/>
+        &nbsp; &nbsp; Add '{"src": "../icon/icon.jpg","sizes": "32x32"}' to "icons" field in app/manifest.json file <br/>
+        &nbsp; &nbsp; $ ../crosswalk-app-tools/src/crosswalk-app build <br/>
+        On Windows: <br/>
+        &nbsp; &nbsp; Shared Mode: <br/>
+        &nbsp; &nbsp; &nbsp; &nbsp; $ node .\crosswalk-app-tools\src\crosswalk-app create org.xwalk.test --android-shared <br/>
+        &nbsp; &nbsp; Embedded Mode: <br/>
+        &nbsp; &nbsp; &nbsp; &nbsp; $ node .\crosswalk-app-tools\src\crosswalk-app create org.xwalk.test <br/>
+        &nbsp; &nbsp; $ cd org.xwalk.test <br/>
+        &nbsp; &nbsp; Add '{"src": "../icon/icon.jpg","sizes": "32x32"}' to "icons" field in app/manifest.json file <br/>
+        &nbsp; &nbsp; $ node  ..\crosswalk-app-tools\src\crosswalk-app build <br/>
+        Shared Mode: <br/>
+        &nbsp; &nbsp; Install shared mode app on android device which is "arm" architecture <br/>
+        &nbsp; &nbsp; Install shared mode app on android device which is "x86" architecture <br/>
+        Embedded Mode: <br/>
+        &nbsp; &nbsp; Install arm app on android device which is "arm" architecture <br/>
+        &nbsp; &nbsp; Install x86 app on android device which is "x86" architecture <br/>
+        Check if icon.png is used to the icon of web app on android device
```

Comment:
Due to icon.png isn't same with icon.jpg, so we can identify which icon is used when we install the app on android device.